client/src/components/pages/application/Signup/index.js

```diff
@@ -1,5 +1,195 @@
 import React from 'react';
+import axios from 'axios';
+import Alert from '../../../common/Alert';
```

Comment:
Error, Can't find file Alert , because you didn't create this component yet, so please delete it for now.  or if you want to add Alert component to this pull request to be easier for you , its ok do that but don't forget to assign comments to its issue that you created recently.